### ISSUE TITLE
Allow PATCH HTTP Method

### ISSUE THREAD

**Issue by dubirajara** [scanapi/scanapi#77]
Hi, I see that HTTP requests PATCH method are not implemented, for any particular reason?

Thank you. 
Congratulations on the project, it is a great idea. 👌

**Comment by camilamaia** [comment]
Hi @dubirajara, thank you very much for the feedback! 😍 

The reason is lack of time 😅 . In theory I believe it would work with `PATCH`, cause we use a generic `.request` method to call the requests:

https://github.com/camilamaia/scanapi/blob/master/scanapi/requests_maker.py#L38

I did not add it to the doc because it would be better to check if it works properly indeed and also to create tests.

If you want to test it or create a PR for it, let me know. I can help you if you need anything 😉

**Comment by camilamaia** [comment]
And BTW, thank you for creating the issue! @dubirajara

**Comment by dubirajara** [comment]
Thank you very much @camilamaia , I will do tests and give you feedback.

**Comment by dubirajara** [comment]
checked,  not works with PATCH:
`ERROR:scanapi.requests_builder:Error to make request `get_entity`. HTTP method not supported: PATCH. Supported methods: ('GET', 'POST', 'PUT', 'DELETE').`

I will do more tests and create a PR for it.

BTW, i tested with v0.0.14 versión, i had problem to run the command **scanapi** with actual version, this error:
```
    from scanapi.tree.api_tree import APITree
ModuleNotFoundError: No module named 'scanapi.tree'
```

thank you.

**Comment by camilamaia** [comment]
@dubirajara 

First, about the problem running scanapi, I created a issue for it and I will investigate it ASAP: https://github.com/camilamaia/scanapi/issues/82

Regarding of the patch method, this message is showing because I added a verification here:
https://github.com/camilamaia/scanapi/blob/master/scanapi/requests_maker.py#L35. I wanted to ensure only tested methods were being using. You can remove this line to check if it works for `patch`

**Comment by camilamaia** [comment]
@dubirajara `ModuleNotFoundError` fixed, please use the version `0.0.17` 😉

**Comment by dubirajara** [comment]
i tested a lot with this method, but not create tests. https://github.com/scanapi/scanapi/pull/113
I have seen that there are no tests for put, post or delete methods.

Thanks for your work.

**Comment by camilamaia** [comment]
No problem, I can add test for PATCH when I add for the others. Thank you very much for the contribution!

**Comment by camilamaia** [comment]
Closed by #113
